.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,48 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
```

Comment:
Should we allow CI to run when not submitted to master?

Reply:
Same as above.